test/blocks/dc-converter-widget/dc-converter-widget.test.js

```diff
@@ -0,0 +1,34 @@
+import { readFile } from '@web/test-runner-commands';
+import { expect } from '@esm-bundle/chai';
+import sinon from 'sinon';
+import { waitForElement, delay } from '../../helpers/waitfor.js';
+
+document.body.innerHTML = await readFile({ path: './mocks/body.html' });
+const { default: init } = await import(
+  '../../../acrobat/blocks/dc-converter-widget/dc-converter-widget'
+);
+
+describe('dc-converter-widget block', () => {
+  before(() => {
+    const block = document.body.querySelector('.dc-converter-widget');
+    init(block);
+  });
+
+  it('shoud creates a DC converter widget block', async function () {
+    this.timeout(5000); // default 2000ms
```

Comment:
It seems we have error on timeout
<img width="1067" alt="image" src="https://user-images.githubusercontent.com/55804872/227075139-958a47a5-a534-4333-a0a1-aeb9676f44a8.png">

I think we may want to use delay() instead.

Reply:
The block is depending on the dev widget inside Adobe network.

Reply:
I added skip for now. Need to test code coverage upload to codecov.io.